Data Structure/Dic.ipynb

```diff
@@ -0,0 +1,177 @@
+#Step1: Download the txt file from mentioned google drive link and save txt file in the same folder where you will save the code of this repo.
+#       Download txt file from here https://drive.google.com/file/d/1MaH727YVLGzEmws2qi9Xk8LLZQ-aL32w/view?usp=sharing
+#
+#Step2: Create .py file and paste the code. and then run it
+#
+#Note:
+#    This is only code which will fetch data (words and their meanings) from txt file and saves each word with its meaning in a node of Linkedlist,
+#    and will create number of nodes depending upon words in txt file.
+
```

Comment:
This is file is attached with repo but in case it is not showing to you please do download it from google drive link